webcompat/form.py

```diff
@@ -48,32 +52,42 @@
 username_message = u'A valid username must be {0} characters long'.format(
     random.randrange(0, 99))
 
+desc_label = 'Please describe what was wrong <span class="wc-Form-required">' \
+    '*</span>'
+desc_message = u'An issue description is required.'
+
 problem_label = (u'What seems to be the trouble?',
                  '<span class="wc-Form-required">*</span>')
 url_label = u'Site URL <span class="wc-Form-required">*</span>'
 
-desc_default = u'''1. Navigate to: Site URL
-2. …
+steps_default = u'''1. I've tried to log in.
+2. I've filled out the form details.
+3. I clicked on the submit button.
+4. Nothing happened.
 
-Expected Behavior:
-
-Actual Behavior:
 '''
 
 
 class IssueForm(FlaskForm):
     '''Define form fields and validation for our bug reporting form.'''
     url = StringField(url_label,
                       [InputRequired(message=url_message)])
-    browser = StringField(u'Browser / Version', [Optional()])
+    browser = StringField(u'Is this information correct?', [Optional()])
     os = StringField(u'Operating System', [Optional()])
     username = StringField(u'Username',
                            [Length(max=0, message=username_message)])
-    description = TextAreaField(u'Give more details', [Optional()],
-                                default=desc_default)
+    description = StringField(desc_label,
+                              [InputRequired(message=desc_message)])
+
+    steps_reproduce = TextAreaField(u'How did you get there?', [Optional()],
+                                    default=steps_default)
     problem_category = RadioField(problem_label,
                                   [InputRequired(message=radio_message)],
                                   choices=problem_choices)
+    browser_test = RadioField(problem_label,
```

Comment:
@karlcow I think he is refering to the WTForm RadioField. see https://wtforms.readthedocs.io/en/latest/fields.html#field-definitions
(scroll down a couple paragraphs to the base class where the constructor takes a label parameter). but to me it seems label is optional since there's a default None value (?).

ex RadioField: https://github.com/webcompat/webcompat.com/blob/master/webcompat/form.py#L74

in the current PR, I also removed the problem_label parameter and I didn't see any issue in the form display or ability to save the issue (and see saved data). but I don't know if there were backend impacts.